<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>258</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="leMsg"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btnSend">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labName">
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="leName"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labAddr">
      <property name="text">
       <string>服务器地址：</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="leAddr"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labPort">
      <property name="text">
       <string>端口：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="lePort"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btnJoinRoom">
      <property name="text">
       <string>进入聊天室</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
